src/workflow/workflow.js

```diff
@@ -302,8 +306,10 @@ const setInstanceMetadata = async (wfConfig) => {
 
   if (
     !instanceIdColumn.records ||
-    !instanceIdColumn.records[startRecordNo] ||
-    !instanceIdColumn.records[endRecordNo - 1]
+    (!instanceIdColumn.records[startRecordNo] &&
+      instanceIdColumn.records[startRecordNo] !== 0) ||
+    (!instanceIdColumn.records[endRecordNo - 1] &&
+      instanceIdColumn.records[endRecordNo - 1] !== 0)
```

Comment:
```suggestion
    instanceIdColumn.records[startRecordNo] === undefined ||
    instanceIdColumn.records[endRecordNo - 1] === undefined
```

Reply:
This will not consider the cases that they are empty strings.

Reply:
We expect to see instanceIds as numbers so maybe I just validate there is a number in that col.

Reply:
> This will not consider the cases that they are empty strings.

Oops, I missed that case.

Yes, let's validate whether it's a number